packages/react-core/src/components/Dropdown/DropdownToggle.tsx

```diff
@@ -30,8 +30,10 @@ export interface DropdownToggleProps extends React.HTMLProps<HTMLButtonElement>
   isDisabled?: boolean;
   /** Whether or not the dropdown toggle button should have primary button styling */
   isPrimary?: boolean;
+  /** An image to display within the dropdown toggle */
+  icon?: React.ReactNode;
   /** The icon to display for the toggle. Defaults to CaretDownIcon. Set to null to not show an icon. */
```

Comment:
Maybe mention indicator in the description to differentiate form the toggleIcon.

Reply:
Updated description